driver-tests/osgi/shaded/pom.xml

```diff
@@ -0,0 +1,159 @@
+<!--
+
+    Copyright DataStax, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.datastax.cassandra</groupId>
+        <artifactId>cassandra-driver-tests-osgi</artifactId>
+        <version>3.11.5-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cassandra-driver-tests-osgi-shaded</artifactId>
+    <name>DataStax Java Driver for Apache Cassandra Tests - OSGi - Shaded</name>
+    <description>A test for the shaded DataStax Java Driver in an OSGi container.</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
+            <classifier>shaded</classifier>
```

Comment:
Am I correct in saying this line is the entire rationale for having this shaded/unshaded/common split?

```
$ diff shaded_pom.xml unshaded_pom.xml 
28,30c28,30
<     <artifactId>cassandra-driver-tests-osgi-shaded</artifactId>
<     <name>DataStax Java Driver for Apache Cassandra Tests - OSGi - Shaded</name>
<     <description>A test for the shaded DataStax Java Driver in an OSGi container.</description>
---
>     <artifactId>cassandra-driver-tests-osgi-unshaded</artifactId>
>     <name>DataStax Java Driver for Apache Cassandra Tests - OSGi - Unshaded</name>
>     <description>A test for the unshaded DataStax Java Driver in an OSGi container.</description>
36d35
<             <classifier>shaded</classifier>
57c56
<                         <id>copy-common-sources</id>
---
>                         <id>copy</id>
76c75
<                         <id>copy-common-test-sources</id>
---
>                         <id>copy-test</id>
```

The other differences look to be either differences in text strings or name of execution descriptors.  Near as I can tell the classifier is the only distinction between these two pom.xml files.  Do I have that right?

Reply:
correct. I should fix up the `maven-resources-plugin` execution ids so they match. I'll also add a comment above the shaded classifier to make it clear that is the material difference